static/js/chart/draw.ts

```diff
@@ -616,16 +509,18 @@ function drawLineChart(
         .data(dataset)
         .enter()
         .append("circle")
-          .attr("class", "dot")
-          .attr("cx", (d, i) => xScale(d[0]))
-          .attr("cy", d => yScale(d[1]))
-          .attr("fill", COLORS[i])
-          .attr("r", 3);
+        .attr("class", "dot")
```

Comment:
could you keep the indent for the .attr please?

Reply:
i think the git UI hides some lines in there... it has extra indents